app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -166,6 +170,21 @@ def create
     changed_value.nil? ? 'None' : changed_value
   end
 
+  private def push_previous_changes_to_changes(previous_changes)
```

Comment:
The name of this method is no longer accurate after your recent changes.

Reply:
Oh ya, changed